cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -1066,6 +1075,60 @@ GoalsAndRequirements getGoalsAndRequirements(HttpServletRequest request,
     }
   }
 
+  private void getUserTaskState(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<UserTaskManager.UserTaskInfo> activeUserTasks = _userTaskManager.getActiveUserTasks();
+    List<UserTaskManager.UserTaskInfo> completedUserTasks = _userTaskManager.getInactiveUserTasks();
```

Comment:
Would the following name change help us improve clarity:
`_userTaskManager.getInactiveUserTasks();` -> `_userTaskManager.getCompletedUserTasks();`